### PR TITLE
scaffold numbering

### PR DESCRIPTION
**Pull request by Alex-Jordan**
This is a companion PR for a `pg` PR that is coming up.

### CODE REVIEW

**Comment by drdrew42** [comment]
Linked PR openwebwork/pg#579

**Review by drgrice1 — CHANGES_REQUESTED** [review]
There are a few changes needed to make this look better.

File: htdocs/js/apps/Scaffold/scaffold.css [hunk 1]
```diff
@@ -65,6 +71,7 @@
 .section-div > .accordion-body div.accordion-inner {
 	background: #FAFAFA;
 	padding: 1rem 1.25rem;
+	padding-right: 0;
```
[the remark this answers is not in the document]
Comment (Alex-Jordan) [reply htdocs/js/apps/Scaffold/scaffold.css:74]: I lost mental track of this and didn't see it yesterday while working on the pg counterpart. Will tend to this soon.

**Comment by Alex-Jordan** [comment]
With @drgrice1's styling commit merged now, this is ready for a review alongside openwebwork/pg#579

**Review by drgrice1 — APPROVED** [review]
This is okay.

**Pull request merged**